lib/pure/httpcore.nim

```diff
@@ -97,6 +97,7 @@ const
   Http504* = HttpCode(504)
   Http505* = HttpCode(505)
 
+const cl* = "\c\L"
```

Comment:
This will be exported to anyone who imports httpclient, because of that this is too short. Rename to ``httpNewLine`` please.